server/athenian/api/slogging.py

```diff
@@ -0,0 +1,281 @@
+import argparse
+import codecs
+import datetime
+import functools
+import io
+import json
+import logging
+import os
+import re
+import sys
+import threading
+import traceback
+from typing import Callable, Dict, Sequence, Tuple, Union
+
+import numpy
+import xxhash
+import yaml
+
+
+logs_are_structured = False
+
+
+def get_datetime_now() -> datetime:
+    """Return the current UTC date and time."""
+    return datetime.datetime.now(datetime.timezone.utc)
+
+
+def get_timezone() -> Tuple[datetime.tzinfo, str]:
+    """Discover the current time zone and it's standard string representation."""
+    dt = get_datetime_now().astimezone()
+    tzstr = dt.strftime("%z")
+    tzstr = tzstr[:-2] + ":" + tzstr[-2:]
+    return dt.tzinfo, tzstr
+
+
+timezone, tzstr = get_timezone()
+_now = get_datetime_now()
+if _now.month == 12:
+    _fest = "🎅"
```

Comment:
Priceless Easter eggs 😍

Reply:
Log parsers would be very happy :grin:

Reply:
Log parsers should always be happy because we will run structured logging in production :)